codespell_lib/data/dictionary.txt

```diff
@@ -4216,12 +4220,18 @@ convinience->convenience
 convinient->convenient
 conviniently->conveniently
 convserion->conversion
+conyak->cognac
+coo->coup
 coodinate->coordinate
 coodinates->coordinates
 coodrinate->coordinate
 coodrinates->coordinates
+cooger->cougar
+cookoo->cuckoo
+coolot->culottes
```

Comment:
Should this not be the singular, and add the plural?

Reply:
Done